L1Trigger/L1TGlobal/src/GlobalBoard.cc

```diff
@@ -310,10 +316,33 @@ void l1t::GlobalBoard::receiveCaloObjectData(const edm::Event& iEvent,
 		  }
 */
 
-        }  //end loop over jet in bx
+        }  //end loop over EtSum objects in bx
       }    //end loop over Bx
     }
   }
+
+  if (receiveZdcEtSums) {
+    edm::Handle<BXVector<l1t::EtSum>> etSumData;
+    iEvent.getByToken(zdcEtSumInputToken, etSumData);
+
+    if (!etSumData.isValid()) {
+      if (m_verbosity) {
```

Comment:
My understanding is that what it is needed by HLT is the uGT emulator output, so it means that the [L1TGlobalProducer](https://github.com/cms-sw/cmssw/blob/master/L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc#L599-L611) when receiving the Calo Object Data (`receiveCaloObjectData`) will get there, exactly. What happens in case an invalid InputTag is provided, I don't know... As far as I can tell, at the moment the code is not breaking (and I am unsure if we are providing a valid but somehow wrong input tag, or an invalid input tag).

Reply:
Even if full verbosity were enabled, the logs from the HLT are basically ignored, so nobody would see the message except maybe during testing.

Reply:
But anyway this is the same approach used for all other collections...